service/src/test/java/bio/terra/cbas/dao/TestRunDao.java

```diff
@@ -12,7 +12,11 @@
 import java.util.UUID;
 import java.util.stream.Collectors;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.springframework.boot.test.context.SpringBootTest;
 
+@SpringBootTest
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Are these annotations needed?

Reply:
Yes. I wish I could tell you why, but without them the tests fail to run.